src/streams.rs

```diff
@@ -417,13 +417,6 @@ pub struct StreamKey {
     pub ids: Vec<StreamId>,
 }
 
-impl StreamKey {
-    /// Return only the stream `id`'s without their data.
-    pub fn just_ids(&self) -> Vec<&String> {
```

Comment:
I actually just pointed this out because users can do `self.ids.iter().map(|msg| &msg.id)` themselves, which is often more effecient since the `Vec` allocation can be avoided.

But hey, whatever works :)